spec/factories/category.rb

```diff
@@ -1,3 +1,5 @@
 FactoryBot.define do
-  factory :category
+  factory :category do
+    title { "Category #{SecureRandom.uuid}" }
```

Comment:
I'm having trouble with unique id for categories (especially on old features spec) so this should work for now.

Reply:
That sounds fine to me.